README.md

```diff
@@ -43,13 +43,14 @@ install ModelBench as a CLI tool or library to use in your own projects.
 ```shell
 pipx install poetry
 ```
+Please be sure to install Poetry in a dedicated python virtual environment separate from the environment used for this project.
```

Comment:
This is a surprise to me. Doesn't pipx create its own environment?

If it is something you want people to do, you'll have to give more detail.

Reply:
Pipx does create its own virtual environment(s) for everything you install in it.